### DISCUSSION TITLE
I‘ve just designed AWM，can we add it？

### DISCUSSION THREAD

**Discussion by chenpishui12345** [HasangerGames/suroi#302] [Ideas]
![捕获](https://github.com/HasangerGames/suroi/assets/165793345/3e6c6eb8-b64a-4bcf-bd08-be8a7f880fee)

**Comment by emeraldn30** [comment]
wow this is good

**Comment by emeraldn30** [comment]
however you need to read the Suroi art guide https://wiki.suroi.io/art_guides/guns

**Reply by chenpishui12345** [reply]
oops

**Comment by emeraldn30** [comment]
weapons should be drawn from the right side among other things

**Comment by HughJassss** [comment]
wait I can see myself in the #patch-notes in the suroi server

**Comment by HughJassss** [comment]
LMAO

**Comment by AshMyBoi** [comment]
Looks good.
